app/views/shared/summary_cards/_group.html.slim

```diff
@@ -21,13 +21,18 @@
           = humanize_date(resource.published_at)
       .summary-card__metadata
         p
+          span.glyphicon.glyphicon-comment.glyphicon--right
+          span.disqus-comment-count data-disqus-identifier="#{url_for(resource)}" 
+            | 0 Comments
```

Comment:
@ptrikutam Just spent some time trying to figure it out, and I can't find why it's not working (and it's starting to piss me off :)). I read countless posts and tried many solutions, but it either doesn't work or just show `1 comment` (even when the resource has more). 

My last hope is that it might be related to the fact that I'm using the staging URL in the configuration (can't put localhost). We might have to try in staging and see how it goes (really don't think it's going to help, but I'm running out of ideas...).